projects/valtimo/plugin/src/lib/plugins/zaken-api/components/create-zaak/create-zaak-configuration.component.html

```diff
@@ -83,6 +83,76 @@
           [tooltip]="'zaakTypeSelectTooltip' | pluginTranslate: obs.pluginId | async"
         ></v-select>
       </ng-container>
+
+      <div *ngFor="let property of propertyList">
+        <ng-container [ngSwitch]="property">
```

Comment:
Use @switch, check this for reference: https://angular.dev/guide/templates/control-flow